input/fsh/profiles.fsh

```diff
@@ -125,10 +125,12 @@ Description: "This profile is to record prescribed ARV regimen against a given t
 * encounter 1..1
 * period 1..1
 * activity 1..* 
+* activity.outcomeCodeableConcept from VSCarePlanActivityOutcome (required)
 * activity.detail 1..1
 * activity.detail.kind = #MedicationRequest
 * activity.detail.code from VSARVMedicationRequest (required)
 * activity.detail.status 1..1
+* activity.detail.scheduledPeriod MS
```

Comment:
@Xhanti2309  Reason for using "MS": This field is only applicable in the case of a Refuse scenario, that is why it is MS, else it is the same as base.  (there are two examples using the same profile for Careplan)